src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -363,18 +351,11 @@ private boolean isStatusMatched(Webhook webhook) {
                 || (result == Result.UNSTABLE && webhook.isNotifyUnstable()));
     }
 
-    private void addScmDetails(List<Fact> factsList) {
+    private void addScmDetails() {
```

Comment:
@casz I had conflict with your change around here so take a look at this, please